From 5bc6b02f9327eeb74c5770e2c1fef6dcd11afa86 Mon Sep 17 00:00:00 2001
From: Pierre Ossman <ossman@cendio.se>
Date: Thu, 6 Oct 2022 16:31:31 +0200
Subject: [PATCH 09/21] Retroactively update NEWS for 0.9.23

We retroactively tagged 0.9.23, so we could not get NEWS properly
included in to it.
---
 NEWS | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/NEWS b/NEWS
index 739df76..5cd9a47 100644
--- a/NEWS
+++ b/NEWS
@@ -22,6 +22,10 @@ Version 0.9.23
   - include AFS support for better serving of files from AFS filesystems
   - use svc_getreq_poll() when available
   - fix serving files larger than 4GB from Windows
+  - better support for 64-bit systems
+  - fixes for macOS and Windows support
+  - fixes when cross compiling
+  - better support for exporting FAT filesystems
 
 Version 0.9.22
 
-- 
2.42.0

